incubator/hnc/test/e2e/demo_test.go

```diff
@@ -43,6 +43,9 @@ var _ = Describe("Demo", func() {
 		// make acme-org the parent of team-a, and team-a the parent of service-1.
 		MustRun("kubectl hns set", nsTeamA, "--parent", nsOrg)
 		MustRun("kubectl hns set", nsService1, "--parent", nsTeamA)
+		// verify
```

Comment:
Can you comment why this is needed for _functional_ correctness and not just because you want to add an additional check? E.g. "wait for HNC to pick up the changes"

Reply:
done